config/kubesan/openshift/patch_priority_class.yaml

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+- op: add
+  path: /spec/template/spec/priorityClassName
+  value: openshift-user-critical
```

Comment:
Is this the standard priority class we use for all storage operators?

Reply:
afaik, yes